app/src/main/java/org/systers/mentorship/view/fragments/RelationPagerFragment.kt

```diff
@@ -5,6 +5,7 @@ import androidx.lifecycle.ViewModelProviders
 import android.os.Bundle
 import com.google.android.material.snackbar.Snackbar
 import android.view.View
+import kotlinx.android.synthetic.main.activity_main.*
```

Comment:
What is the use of adding this import statement here?
Remove if not required.

Reply:
I have removed it in my new commit.